src/treemodel/delta/mergedelta.js

```diff
@@ -47,6 +48,10 @@ export default class MergeDelta extends Delta {
 	get _reverseDeltaClass() {
 		return SplitDelta;
 	}
+
+	static get className() {
```

Comment:
As proposed recently, let's document all inherited props and methods with `/** @inheritDoc */`.